exasol_bucketfs_utils_python/buckets.py

```diff
@@ -0,0 +1,36 @@
+from exasol_bucketfs_utils_python import BucketFsError
+import requests
+from requests.auth import HTTPBasicAuth
+from typing import Generator
+
+
+def list_buckets(
+    base_url: str,
+    path: str = "",
+    username: str = None,
+    password: str = None,
+    port: int = 6666,
```

Comment:
No, and we shouldn't specify a default in my opinion

Reply:
@tkilias I think having a default port is good but it should be the default used for the service (which in this case seems to be `2580` rather than `6666`).

**Why I think it's good?** (**_TL;DR: convenience &  simplicity_**)
* If the service anyhow has a default port it simplifies the API usage for the common case
* It's great having the flexibility to control each detail but most API users rather go with
   `api = Api()` rather than `api = Api(token='foo', name='bar', port=5555, ip='127.0.0.1', ....)`
   at least until they have more complex use cases.